diff --git a/0x03-git/bash/alx b/0x03-git/bash/alx
index eaa5489..f144a8d 100644
--- a/0x03-git/bash/alx
+++ b/0x03-git/bash/alx
@@ -1,2 +1,2 @@
 #!/bin/bash
-echo "ALX School is so cool!"
+echo "ALX"
diff --git a/0x03-git/js/README.md b/0x03-git/js/README.md
new file mode 100644
index 0000000..52c897f
--- /dev/null
+++ b/0x03-git/js/README.md
@@ -0,0 +1 @@
+new readme
diff --git a/0x03-git/js/index.js b/0x03-git/js/index.js
new file mode 100644
index 0000000..e69de29
diff --git a/0x03-git/js/main.js b/0x03-git/js/main.js
new file mode 100644
index 0000000..e69de29
diff --git a/0x03-git/up_to_date b/0x03-git/up_to_date
deleted file mode 100644
index d0aed61..0000000
--- a/0x03-git/up_to_date
+++ /dev/null
@@ -1 +0,0 @@
-git pull
diff --git a/up_to_date b/up_to_date
deleted file mode 100644
index d0aed61..0000000
--- a/up_to_date
+++ /dev/null
@@ -1 +0,0 @@
-git pull
